Facebook.iOS/source/AudienceNetwork/ApiDefinition.cs

```diff
@@ -776,20 +810,24 @@ interface InstreamAdView {
 		AdExtraHint ExtraHint { get; set; }
 
 		// -(instancetype _Nullable)initWithPlacementID:(NSString * _Nonnull)placementID __attribute__((objc_designated_initializer));
+		[Obsolete("Instream ads have been deprecated. Initialiser will return nil.")]
```

Comment:
Typo in the en-US right? `Initialiser`-> `initializer`
```suggestion
		[Obsolete("Instream ads have been deprecated. initializer will return nil.")]
```

If yes, same for the ones below

Reply:
Apparently it is an acceptable alternate spelling?  (says Google)